package/libyui-qt-doc.spec

```diff
@@ -21,7 +21,7 @@
 
 Name:           %{parent}-doc
 # DO NOT manually bump the version here; instead, use   rake version:bump
-Version:        2.52.0
+Version:        2.52.1
```

Comment:
to be honest I am a bit lost about that API/ABI incompatible changes and versioning. So lets hope that increase of patch level is enough.

Reply:
That is indeed a problem with libyui.

I would have increased the minor version of the library but that does not work in the libyui setup (see comment in VERSION.cmake).